src/jshint.js

```diff
@@ -855,7 +844,7 @@ var JSHINT = (function() {
 
     // if current expression is a let expression
     if (!initial && state.tokens.next.value === "let" && peek(0).value === "(") {
-      if (!state.option.inMoz(true)) {
+      if (!state.inMoz(true)) {
```

Comment:
Since all occurrences have to be touched in this patch, maybe also refactor this Boolean arg into an options object?